invenio_accounts/views/settings.py

```diff
@@ -54,7 +54,7 @@ def init_menu():
     item = current_menu.submenu('breadcrumbs.settings')
     item.register('invenio_userprofiles.profile', _('Account'))
 
-    ### Register menu ###
+    # # # Register menu # # #
```

Comment:
this was a pycodestyle issue

Reply:
oh wow! :)

You can also change it as the one above `# - Register menu`